aten/src/ATen/core/ivalue_inl.h

```diff
@@ -191,6 +191,7 @@ struct C10_EXPORT ivalue::Future final : c10::intrusive_ptr_target {
   }
 
  public:
+  Future();
```

Comment:
Every future needs a type, even if it is `NoneType::get()` that matches the type of the value eventually placed in the future. The uses of future should be changed to pass a type. Adding a default constructor will lead people to add the wrong type or forget to add one.

Reply:
👍